docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,63 @@
+.NET Core
+=========
+
+.NET Core is a general purpose development platform developed by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supports Windows, OS X and Linux and intends to support a wide variety of CPUs and OSes over time. It can be used in device, cloud and embedded/IoT scenarios. We expect it to be extended to support additional scenarios over time (e.g. WebAssembly), both by Microsoft and the community.
```

Comment:
OSes -> operating systems (I think it's better to spell out. I also couldn't find this on our terminology database)
e.g. -> for example